conf/evolutions/default/85.sql

```diff
@@ -1,6 +1,12 @@
 # --- !Ups
 ALTER TABLE `users`
-ADD COLUMN `passwordCreatedDate` DATETIME NOT NULL AFTER `password`;
+
+ADD COLUMN `passwordCreatedDate` DATETIME NOT NULL AFTER `password`,
+ADD COLUMN `date_created` DATETIME NOT NULL AFTER `last_login`,
+ADD COLUMN `created_by` INT NOT NULL AFTER `creation_date`;
+
+UPDATE `users`
+SET date_created = '9999-01-01'
 
 UPDATE `users`
 SET passwordCreatedDate = '0000-01-01';
```

Comment:
I had issues with this evolution running correctly. There is a semi-colon missing at line 9, also I figured the default dates should probably be consistent.

UPDATE `users`
SET date_created = '0000-01-01 00:00:00';

UPDATE `users`
SET passwordCreatedDate = '0000-01-01 00:00:00';